tvtk/pyface/tvtk_scene.py

```diff
@@ -808,7 +816,7 @@ def _create_control(self, parent):
 
         self._interactor.initialize()
         self._interactor.render()
-        self.light_manager = light_manager.LightManager(self)
+        self.light_manager = light_manager.LightManager(weakref.proxy(self))
```

Comment:
@dmsurti, Please note that this fix removes the reference cycles which are collectible. There is no memory leak here. The cycle between `Scene` and `LightManager` will however keep the `Scene` alive if the `LightManager` is referenced by other components an application.